todo.md

```diff
@@ -113,6 +94,15 @@
       1. the _base_ checkout depends on _cart_
       2. the _b2b_ / _repl_ depend on _base_
       3. when landing on a _b2b_ step, will the order of chunks being loaded be the following: _cart_ first, then _base_, and lastly the _b2b_ chunk?
+8. There's a TS error on the last checkout step (review order) when running b2b. Seems related to import-export. (fix existing issue from develop)
```

Comment:
moved this to near the end as this error is the same behavior from develop

Reply:
I think we can remove it altogether, if the issue is present on develop.